apps/browser/qml/pages/components/OverlayListItem.qml

```diff
@@ -24,7 +24,7 @@ BackgroundItem {
     Item {
         id: iconContainer
 
-        width: iconWidth
+        width: iconWidth + horizontalOffset
```

Comment:
The indent from the left edge will increase by `horizontalOffset + horizontalOffset/2`  so and supposed?

Reply:
It will increase, yes, to match the indent used by the addTabButton in the SecondaryBar
See: https://github.com/sailfishos/sailfish-browser/blob/master/apps/browser/qml/pages/components/SecondaryBar.qml#L83

Or, have I made some mistake (I don't have tablet to test on, so I might be missing something in that case?)